tools/pipelines/build-docs.yml

```diff
@@ -49,17 +50,20 @@ variables:
     value: ${{ or(variables.releasePipeline, variables.n1Pipeline, variables.latestPipeline) }}
   - name: Packaging.EnableSBOMSigning
     value: true
-  - name: isMain
-    value: ${{ eq(variables['Build.SourceBranchName'], 'main') }}
+  - name: isMainOrNext
```

Comment:
@tylerbutler Does this look correct?  Is this something we want to trigger out of both or should this remain a "main"-only thing since that remains the source of all the release branches?